services/QuillLMS/app/models/district.rb

```diff
@@ -19,8 +19,14 @@
 #  clever_id      :string
 #  nces_id        :integer
 #
+# Indexes
+#
+#  index_districts_on_nces_id  (nces_id) UNIQUE
+#
 class District < ApplicationRecord
 
+  validate :validate_name_not_blank
```

Comment:
I think the ActiveRecord validation [presence](https://guides.rubyonrails.org/active_record_validations.html#presence) would suffice: `validates :name, presence: true`?

Then you can get rid of the `validate_name_not_blank`